frontend/src/utilities/EffectUtility.ts

```diff
@@ -35,6 +35,6 @@ export default class EffectUtility {
       return response;
     }
 
-    return !Array.isArray(response.data) ? new Model(response.data) : (response.data.map((json) => new Model(json)) as any);
+    return !Array.isArray(response.data) ? new Model(response.data.data) : (response.data.map((json) => new Model(json)) as any);
```

Comment:
We can just this on the api side so its not too nested with the response that comes back.